api/src/server.js

```diff
@@ -163,7 +163,7 @@ export const bootPlugin = ({ getState, setState }) => {
             case 'encouragement/getEncouragement': {
               send({
                 type: 'encouragement/getEncouragementResponse',
-                data: await E(publicFacet).getFreeEncouragement(),
+                data: await E(publicFacet).getFreeEncouragement(obj.nickname),
```

Comment:
```suggestion
                data: await E(publicFacet).getFreeEncouragement(obj.data.nickname),
```
I could misunderstanding, but I think this should have the same data structure as `encouragement/sendInvitation`

Reply:
Just tested locally, and the original seems to work. I think users will find the different confusing, or at least I do :)

Reply:
> I could misunderstanding, but I think this should have the same data structure as `encouragement/sendInvitation`

Good catch!  I fixed this.